components/eamxx/scripts/atmchange

```diff
@@ -66,7 +73,14 @@ OR
         help="Used by buildnml to replay buffered commands",
     )
 
-    parser.add_argument("changes", nargs="+", help="Values to change")
+    parser.add_argument(
+        "--reset",
+        default=False,
+        action="store_true",
+        help="Forget all previous atmchanges",
```

Comment:
But does not reset the XML to whatever case.setup would generate, right? I think of `--reset` as restoring the default/original values, while that's not happening here.

Reply:
This came up during the v1 telecon.  With the recent change to our buildnml which buffers all `atmchange` calls into an XML entry it isn't easy to reset that buffer to be empty if you want to return to defaults.  So this option is supposed to take care of that.

Reply:
Ah, nvm, I thought we did not re-generate the XML file, but Jim's last PR on buildnml made it so that the atm xml is always rebuilt. Unless SCREAM_HACK_XML=ON.

@jgfouca should we print a warning if SCREAM_HACK_XML=ON  and --reset is used? As things stand, resetting the buffer won't do anything if SCREAM_HACK_XML=ON, since the next buildnml will not regenerate the XML.